### PR TITLE
Removed the github tooling helper

### PR DESCRIPTION
**Pull request by adambraimbridge**
Re: https://github.com/Financial-Times/tooling-helpers/issues/58

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -78,3 +78,7 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+# Generated files
+package-lock.json
+test-results/
```
Comment (adambraimbridge) [inline .gitignore]: I forget our convention/policy for whether or not to include `package-lock.json`

File: index.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
 module.exports = {
     git: require('./git/src'),
     PackageJson: require('./package-json/src'),
-    github: require('./github/src')
 };
```
Comment (simonplend) [inline index.js]: Whoops, I accidentally left this file in when I made this repository a monorepo. It can be removed.

File: .gitignore (hunk 1 above)
Comment (adambraimbridge) [reply .gitignore]: Answer: Once we have the right dependency-management in place (Renovate) and configured for all of our repositories, we can make good use of `package-lock.json`. For now, we're ignoring them.

File: index.js (hunk 2 above)
Comment (adambraimbridge) [reply index.js]: Done. Thanks

Comment (adambraimbridge) [reply index.js]: Done. Thanks

**Pull request merged**